### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Configure dependabot to get security updates on the dependencies.

- https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool
- https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

### Alert on my fork
image

#### PR created by dependabot for the vulnerability 
https://github.com/naveensrinivasan/mempool/pull/1

Other than merging this PR the security setting has to be enabled 

image

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
```
Comment (knorrium) [inline .github/dependabot.yml:4]: Can you add explicit separate entries for the backend and frontend directories?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/docker/backend"
+  schedule:
+    interval: daily
+  commit-message:
+      prefix: ":seedling:"
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/docker/electrum"
```
Comment (knorrium) [inline .github/dependabot.yml:21]: This can be removed as we are not building Electrum and will be cleaned up soon

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/"
```
Comment (knorrium) [inline .github/dependabot.yml:9]: Can you scope this down to the backend directory like you did for the frontend?

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/docker/backend"
+  schedule:
+    interval: daily
```
Comment (knorrium) [inline .github/dependabot.yml:16]: The docker schedules can be set to weekly as they are not used frequently

**Review by knorrium — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (naveensrinivasan) [reply .github/dependabot.yml]: I have done that.

File: .github/dependabot.yml (hunk 3 above)
Comment (naveensrinivasan) [reply .github/dependabot.yml]: Addressed it.

File: .github/dependabot.yml (hunk 4 above)
Comment (naveensrinivasan) [reply .github/dependabot.yml]: Addressed it.

File: .github/dependabot.yml (hunk 2 above)
Comment (naveensrinivasan) [reply .github/dependabot.yml]: Addressed it.

File: .github/dependabot.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  directory: "/frontend"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: docker
+  directory: "/docker/backend"
+  schedule:
+    interval: weekly
+- package-ecosystem: "github-actions"
+  directory: "/"
```
Comment (knorrium) [inline .github/dependabot.yml:18]: I forgot to ask to set this to `.github/workflows/`. Can you please update it as well?

Comment (naveensrinivasan) [reply .github/dependabot.yml:18]: This is what is for actions. By default that is the folder, it looks for. I have already addressed it.

**Comment by naveensrinivasan** [comment]
A friendly ping. What does it need to happen to merge this?

**Pull request merged**